components/Accordion/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+
+import './style.scss';
+
+const Accordion = ({ children, icon, title }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleToggle = () => {
+    setIsActive(!isActive);
+  }
+
+  return (
+    <div className="Accordion">
+      <button className="Accordion-title" onClick={handleToggle}>
+        <i className={`Accordion-arrow${isActive ? '_active' : ''} fa fa-chevron-right`}></i>
```

Comment:
It seems _maybe_ more common to have the toggle icon on the right side? And I think that would be less visualizing confusing with the icon?

Reply:
yea it does look a little odd with the icon.. but i think we had already agreed on the right toggle 😅